appsettings/src/tree/AppSettingsTreeItem.ts

```diff
@@ -35,6 +36,15 @@ export function validateAppSettingKey(settings: StringDictionary, client: IAppSe
     return undefined;
 }
 
+export function validateAppSettingValue(value: string, client: IAppSettingsClient): string | undefined {
+    if (client.isContainer && value === (null || undefined || ' ')) {
```

Comment:
Alternative solution:

```typescript
value = value.trim();
if (client.isContainer && !value) {
  // ...
}
```

Also, just for my own clarification, but why do we only care about the value not being empty when the client is container?

Reply:
In the issue they point out it doesn't cause an error for normal function apps just containerized ones. It's likely not something they would want to do but it is possible to hav an empty value for normal function apps.

Reply:
I don't think that it would actually work though:
![image](https://github.com/microsoft/vscode-azuretools/assets/5290572/6d2bdc60-3aaa-4dbc-a64a-014c05909271)

Even if it doesn't, I think it's a stupid thing for users to do and am fine with blocking them on it.

Reply:
Oki doke I'll change it to be for both.